onnxruntime/contrib_ops/cpu/tokenizer.cc

```diff
@@ -249,37 +80,37 @@ Tokenizer::Tokenizer(const OpKernelInfo& info) : OpKernel(info) {
     ORT_ENFORCE(status.IsOK(), "Either one of the separators OR tokenexp attributes required but none is set");
     ORT_ENFORCE(!tokenexp.empty(), "Expecting a non-empty tokenexp");
   } else {
-    ORT_ENFORCE(!separators.empty(), "Expect at least one item within separators");
+    ORT_ENFORCE(!separators.empty(), "separators must not be empty");
+    if (separators.size() == 1) {
+      if (separators[0].empty() || separators[0] == ".") {
```

Comment:
`seperators[0] == "."` would remove everything.

Reply:
My bad

---
In reply to: [277479153](https://github.com/Microsoft/onnxruntime/pull/857#discussion_r277479153) [](ancestors = 277479153)